vmdb/app/controllers/ems_infra_controller.rb

```diff
@@ -17,6 +17,51 @@ def self.table_name
   def index
     redirect_to :action => 'show_list'
   end
+  
+  def scaling
+    assert_privileges("ems_infra_scale")
+    if params[:cancel]
+      redirect_to :action => 'show', :id => params[:id]
+    end
+    
+    drop_breadcrumb(:name => _("Scale Infrastructure Provider"), :url => "/ems_infra/scaling")
+    @infra = EmsOpenstackInfra.find(params[:id])
+    @stack = @infra.orchestration_stacks.first
```

Comment:
please add todo that states this counts with one stack being associated with provider and it should support more in the future